modules/contact/src/constraints/RANFSNormalMechanicalContact.C

```diff
@@ -36,6 +36,10 @@ validParams<RANFSNormalMechanicalContact>()
   params.addClassDescription("Applies the Reduced Active Nonlinear Function Set scheme in which "
                              "the slave node's non-linear residual function is replaced by the "
                              "zero penetration constraint equation when the constraint is active");
+  params.addParam<bool>("ping_pong_protection",
+                        false,
+                        "Whether to protect against ping-ponging by tying the slave node to the "
+                        "edge between the involved master faces");
```

Comment:
What kind of protective equipment do ping pong players wear? Wrist guards? Knee pads? Helmets? Protective eyewear?

Reply:
Haha, I've updated the doc string to be a little more explicit about what ping-ponging is. I guess I could call it oscillation protection...

Reply:
Nah, I think I'd leave it how you have it.